plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeAdlsStorage.java

```diff
@@ -16,10 +16,10 @@
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
 import com.google.common.io.Resources;
-import io.trino.plugin.deltalake.util.DockerizedDataLake;
-import io.trino.plugin.deltalake.util.TestingHadoop;
+import io.trino.plugin.hive.containers.HiveHadoop;
```

Comment:
This test class looks broken now. I can't run locally. Please note that CI doesn't run this test because of forked branch PR.
```
java.lang.IllegalArgumentException: Resource with path /var/folders/8s/dkvf18z55lj_9yxhy1n54sph0000gn/T/core-site16451568450651754371.xml could not be found on any of these classloaders: [jdk.internal.loader.ClassLoaders$AppClassLoader@3b192d32]
```

Reply:
I've tested the class locally by removing the constructor. The container is launched successfully now and I can also execute hadoop fs commands. Let me know if it works on your side.

Reply:
It works now, thanks!